drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -826,14 +805,46 @@ function DragAndDropBlock(runtime, element, configuration) {
         return feedback_msgs_list.map(function(message) { return message.message; }).join('\n');
     };
 
-    var updateDOM = function(state) {
+    var updateDOM = function() {
         var new_vdom = render(state);
         var patches = virtualDom.diff(__vdom, new_vdom);
         root = virtualDom.patch(root, patches);
         $root = $(root);
         __vdom = new_vdom;
     };
 
+    // Uses edX JS accessibility tools to read feedback messages when present.
+    var readScreenReaderMessages = function() {
+        if (window.SR && window.SR.readText) {
```

Comment:
Thank you, that is a very good idea. I implemented the suggested approach, which also allowed me to add `aria-atomic="true"` to the live element to ensure feedback messages are read in entirety each time, even if only part of the feedback changes. I did some testing with VoiceOver and it looks like feedback messages are now always announced correctly.